changelog/fragments/1695289867-Secondary-fallback-for-package-signature-verification.yaml

```
@@ -0,0 +1,31 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: Secondary fallback for package signature verification
+
+# Long description; in case the summary is not enough to describe the change
+# this field accommodate a description without length limits.
+description: Ability to upgrade securely in Air gapped environment where fleet server is only reachable URI.
```

[Suggestion]
I believe a few small changes here will make it easier to understand. 

```suggestion
description: Ability to upgrade securely in Air gapped environment where fleet server is the only reachable URI.
```

or
```suggestion
description: Ability to securely upgrade in Air gapped environment, where the fleet server is the only reachable URI.
```